Files/Helpers/XamlHelpers/DependencyObjectHelpers.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Files.Helpers.XamlHelpers
+{
+    public static class DependencyObjectHelpers
+    {
```

Comment:
Excellent organization here! We want contributors to find these helpers with ease.